tests/main/ubuntu-core-gadget-config-defaults/task.yaml

```diff
@@ -25,8 +26,14 @@ prepare: |
     SUFFIX=
     MODEL=developer1-pc-w-config.model
     if is_core18_system; then
-        SUFFIX=-core18
-        MODEL=developer1-pc-18-w-config.model
+        if [ "$SERVICE" = "rsyslog" ]; then
+            echo "The service to test does not exist in the core18 system, skipping..."
+            touch "${SERVICE}.skip"
+            exit
+        else
```

Comment:
Is `else` needed here? I think it would be cleaner without it (we exit after all).